src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2167,6 +2177,12 @@ Mavlink::task_main(int argc, char *argv[])
 		_main_loop_delay = MAVLINK_MAX_INTERVAL;
 	}
 
+	/* set the instance ID */
```

Comment:
Are these comments really necessary? :wink:

Reply:
The method names are self-documenting. +1 to getting rid of unnecessary comments.

Reply:
I was keeping with the general trend in the lines above/below. Removed!  Thanks for your feedback!